site/en/docs/privacy-sandbox/chips/index.md

```diff
@@ -0,0 +1,139 @@
+
+Currently, third-party cookies can enable services to track users and join their information from across many unrelated top-level sites. This is known as cross-site tracking. 
+
+{% Img src="image/vgdbNJBYHma2o62ZqYmcnkq3j0o1/jLOlPtLY9Zqte4IOoU6g.png", alt="ALT_TEXT_HERE", width="800", height="450" %}
+
+For example, when a user visits site A, embedded content from site C can set a cookie on the user's machine in response to the cross-site request. If the user then visits site B which also embeds C, site C could access the same cookie that has been set earlier when the user visited site A. This allows site C to compile a user's browsing activity across site A, site B, and every site that it is embedded on.
+
+To protect user privacy, browser vendors are placing restrictions on this behavior and phasing out support for third-party cookies.
+
+While cross-site tracking is an issue, there are still valid cross-site cookie use cases on the web today which can be achieved in a privacy-preserving way with cookie partitioning.
+
+### Use cases
+
+For example, the site `retail.example` may want to work with a third-party service `support.chat.example` to embed a support chat box on its site. Many embeddable chat services today rely on cookies to save interaction history. 
+
+{% Img src="image/vgdbNJBYHma2o62ZqYmcnkq3j0o1/jsxgCpkMRXwXughPjg7j.png", alt="ALT_TEXT_HERE", width="800", height="620" %}
+
+Without the ability to set a cross-site cookie, `support.chat.example` could instead rely on `retail.example` passing along their first-party session identifier (or some derived value of it). In that case, every website that `support.chat.example` is embedded on would require additional setup to pass along the state.  
+
+Alternatively, `support.chat.example` may request `retail.example` to embed JavaScript that's hosted on `support.chat.example` on `retail.example` pages. This introduces security risks, because it allows the `support.chat.example` script to have elevated privileges on `retail.example`, such as the ability to access authentication cookies.
+
+Example use cases for CHIPS include any scenarios where cross-site subresources require some notion of session or persistent state that is scoped to a user's activity on a single top-level site, such as:
+
++   Third-party chat embeds
++   Third-party map embeds
++   Subresource CDN load balancing
++   Headless CMS providers
++   Sandbox domains for serving untrusted user content (such as googleusercontent.com and githubusercontent.com)
++   Third-party CDNs that use cookies to serve content that's access-controlled based on the authentication status on the first-party site (for example, profile pictures on social media sites hosted on third-party CDNs)
++   Front-end frameworks that rely on remote APIs using cookies on their requests
+
+### How does it work?
+
+CHIPS introduces a new cookie attribute, `Partitioned`, to support cross-site cookies that are partitioned by top-level context.  
```

Comment:
Consider replacing "partitioned by top-level context" with "divided by top-level context." Since partitioned has been used frequently and not defined in another way, it may be helpful for users to have another way of thinking about that language.